src/main/java/uk/gov/pay/adminusers/utils/email/EmailValidator.java

```diff
@@ -56,7 +56,8 @@
             "ucds.email",
             "wmca.org.uk",
             "york.ac.uk",
-            "digitalaccessibilitycentre.org"
+            "digitalaccessibilitycentre.org",
+            "fermanaghomagh.com"
```

Comment:
Can you put this in the right place alphabetically please? (It’s generally in alphabetical order, except for digitalaccessibilitycentre.org, which I assume is because they were given access for the accessibility audit rather than because it’s a public sector domain.)